google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/CloseStream.java

```diff
@@ -35,8 +37,22 @@ public abstract class CloseStream implements ChangeStreamRecord, Serializable {
 
   private static CloseStream create(
       com.google.rpc.Status status,
-      List<ChangeStreamContinuationToken> changeStreamContinuationTokens) {
-    return new AutoValue_CloseStream(Status.fromProto(status), changeStreamContinuationTokens);
+      List<ChangeStreamContinuationToken> changeStreamContinuationTokens,
+      List<ByteStringRange> newPartitions) {
+    if (status.getCode() == 0) {
```

Comment:
I think you can simplify this

Preconditions.checkState((status.getCode() == 0 && changeStreamContinuationTokens.isEmpty) || (status.getCode() != 0 && !changeStreamContinuationToken.isEmpty())

Reply:
Discussed offline. Let's keep two checks because I want to use different error messages.